src/server/util/array.ts

```diff
@@ -0,0 +1,9 @@
+const arraysContainSame = (a: any[] | undefined, b: any[] | undefined) => {
```

Comment:
Can use the lodash library [_.isEqual](https://lodash.com/docs/#isEqual) instead of building a custom comparator

Reply:
I didn't know this exist, will try it out

Reply:
done